### ISSUE TITLE
question about evaluation partition.

### ISSUE THREAD

**Issue by Cheng-Xue** [megvii-research/DPGN#21]
Hi,
Interesting work and well done!
If I understand the code correctly, to get the best model when training, you used the test set to validate the accuracy. Should we use the validation set instead?

Cheers

**Comment by AlienceGG** [comment]
Hi Cheng-Xue,
 
The issue #8 (eg. Q2) might be helpful to you.

Yours 
DPGN team
